### DISCUSSION TITLE
how to change fileList to array

### DISCUSSION THREAD

**Discussion by Raju-2101** [react-hook-form/react-hook-form#2415] [Q&A] [answered]
i have  **`<input type="file" multiple name="images" ref={register}  />`** and it saves it as fileList [{..}, {..}]. i want to change the fileList as array. i tried **setValue("image", [..., ...])** but it doesn't work. is there another way.

**Comment by bluebill1049** [comment, marked as answer]
nope, doesn't work with file input, it's immutable. do at submit level

**Comment by Raju-2101** [comment]
alright.
